usage_examples/streamlit/streamlit_app.py

```diff
@@ -0,0 +1,32 @@
+from spacy import util
+from spacy_streamlit import visualize_ner, visualize_textcat
+import streamlit as st
+
+DEFAULT_TEXT = "Matt Honnibal is the CEO of Explosion."
+
+st.title("spacy-llm Streamlit Demo")
+text = st.text_area("Text to analyze", DEFAULT_TEXT, height=70)
+
+textcat_config = util.load_config(
+    "usage_examples/streamlit/openai_textcat_zeroshot.cfg"
+)
+textcat_model = util.load_model_from_config(textcat_config, auto_fill=True)
+
+ner_config = util.load_config("usage_examples/streamlit/openai_ner_zeroshot.cfg")
+ner_model = util.load_model_from_config(ner_config, auto_fill=True)
+
+models = {"textcat": textcat_model, "ner": ner_model}
+model_names = models.keys()
+
+selected_model = st.sidebar.selectbox("Model", model_names)
+
+nlp = models[selected_model]
+doc = nlp(text)
```

Comment:
I'm wondering how streamlit's reactive process works with this. Does it call `nlp` as you're typing? If yes, it might be costly and it might be better to have this line be enclosed in some "press button" action.

Reply:
I think the default Streamlit text input field needs you to submit before it executes further!

Reply:
Yep, exactly, you need to submit it first 😄